ansys/dpf/core/operators/averaging/elemental_difference.py

```diff
@@ -99,22 +114,23 @@ def outputs(self):
 #internal name: elemental_difference
 #scripting name: elemental_difference
 class InputsElementalDifference(_Inputs):
```

Comment:
I've never seen an input parameter be linked like this. I assume it is OK and that no "Parameters" section with  description is needed here.